src/wait-for-element.js

```diff
@@ -62,4 +54,11 @@ function waitForElement(
   })
 }
 
+function getDocument() {
+  if (typeof document === 'undefined') {
+    throw new Error('Could not find default container')
```

Comment:
No, this is a more elegant way of what you were doing before. To avoid running this code all you have to do is provide a `container` to `waitForElement`. If you don't, it'll throw the error (which is what was happening before).

Reply:
Hmm ok it seems fine, I didn't realize JS evaluates default values lazily